cuqi/randomvariable/_randomvariable.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import annotations
+from typing import List, Any
+from ._ast import RandomVariableNode
+from ._orderedset import OrderedSet
+import operator
+import cuqi
+
+
+class RandomVariable:
+    """ Random variable defined by a base distribution with potential algebraic operations applied to it.
```

Comment:
can you add parameter section for the __init__ parameters `distributions` and `tree`